packages/cerebral-forms/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "cerebral-forms",
+  "version": "1.0.0-alpha.1",
+  "description": "Signals, actions and state factories to create forms",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "../../node_modules/.bin/mocha --compilers js:../../node_modules/babel-register 'tests/**/*.js'",
+    "test:watch": "npm run test -- --watch",
+    "prebuild": "npm run test",
+    "build": "BABEL_ENV=production ../../node_modules/.bin/babel src/ --out-dir=lib/ -s",
+    "coverage": "../../node_modules/.bin/nyc --reporter=lcov --reporter=text npm run test",
```

Comment:
we need json reporter here:
../../node_modules/.bin/nyc --reporter=lcov --reporter=json npm run test